dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.javax.persistence-2.1.feature

```diff
@@ -3,7 +3,8 @@ symbolicName=com.ibm.websphere.appserver.javax.persistence-2.1
 singleton=true
 IBM-Process-Types: server, \
  client
--features=com.ibm.websphere.appserver.javax.persistence.base-2.1
+-features=com.ibm.websphere.appserver.javax.persistence.base-2.1, \
```

Comment:
Not sure why you are modifying this feature instead of the com.ibm.websphere.appserver.javax.persistence.base features themselves.

Reply:
Noted. I'll follow up on it.